Makefile

```diff
@@ -281,12 +281,12 @@ e2e-stop-instrumented:
 
 .PHONY: e2e-debug
 e2e-debug:
-	kubectl get all -n $(CONTROLLER_NAMESPACE)
-	kubectl get leases -n $(CONTROLLER_NAMESPACE)
-	kubectl get all -n $(WATCH_NAMESPACE)
-	kubectl get certificatepolicies.policy.open-cluster-management.io --all-namespaces
-	kubectl describe pods -n $(CONTROLLER_NAMESPACE)
-	kubectl logs $$(kubectl get pods -n $(CONTROLLER_NAMESPACE) -o name | grep $(IMG)) -n $(CONTROLLER_NAMESPACE)
+	-kubectl get all -n $(CONTROLLER_NAMESPACE)
+	-kubectl get leases -n $(CONTROLLER_NAMESPACE)
+	-kubectl get all -n $(WATCH_NAMESPACE)
+	-kubectl get certificatepolicies.policy.open-cluster-management.io --all-namespaces
+	-kubectl describe pods -n $(CONTROLLER_NAMESPACE)
+	-kubectl logs $$(kubectl get pods -n $(CONTROLLER_NAMESPACE) -o name | grep $(IMG)) -n $(CONTROLLER_NAMESPACE)
```

Comment:
I had to look up what this did, but it's very helpful!

Reply:
Yes! I saw it in the build-harness. In general, it creates a little bit of Makefile noise since there's output from Make that the lines were ignored, but I prefer it to `|| true` and it was annoying when the workflow would skip down to the debug failures when that wasn't the issue.